### ISSUE TITLE
Meetup #2

### ISSUE THREAD

**Issue by betatim** [pydatazrh/meetup#7]
All things related to making the first meetup happen.

## Topics

* **Awesome Title**, by [Your Name-Here](https://example.com)
_This will be the best talk ever._
* **Awesome Title**, by [Your Name-Here](https://example.com)
_This will be the best talk ever._

## Location

TBD. Want to offer a place? Post in this issue.

## Date

Early December 2016, before the Christmas hectic starts.

**Comment by betatim** [comment]
Should we already create a palceholder on meetup.com for the second meetup (and link it)? Gives people a place to volunteer themselves as speakers 😀

**Comment by TitusCornelius** [comment]
yes, if you could do that, it would be great.. otherwise, I might get to it
sometime this weekend..

Great work on finding a location for the first one :D

**Comment by betatim** [comment]
@sebubu does the description of the talk work for you?

How does one of 6, 7, 8 December work for you guys? We will have to play the iteration game between speakers and location 😀

**Comment by Sebubu** [comment]
@betatim If @jaimefrio accepts OSMDeepOD as topic then everything is fine :smile:

**Comment by betatim** [comment]
@metaodi could we use the Arena again on 7 December please?

**Comment by metaodi** [comment]
@betatim unfortunately the Arena is already used for another event on 7 December. Actually the only possible day in this week would be Friday.

**Comment by betatim** [comment]
Ok, we'll shop around and get back to you. Friday doesn't seem ideal for a meetup.

**Comment by uweschmitt** [comment]
@aktion-hip What about organizing a room at ETH ?

**Comment by TitusCornelius** [comment]
I think we should announce the meetup and see from there.. I took a day to
find a room last time after the announcement.. Thanks all for taking the
initiative in organizing this one :)

**Comment by aktion-hip** [comment]
I could try to book a room in the ETH Hauptgebäude on 6/7/8 of December either for max 50 persons or a lecture hall (up to 160 persons).
Should I give it a try?
If yes: What size would you prefer?

**Comment by betatim** [comment]
Yes please! I think 50 people would be enough. The Python meetup will be on the 8th so maybe we strike that date as a possibility.

**Comment by aktion-hip** [comment]
@betatim To be clear: by _strike_, you mean _eliminate_?

**Comment by betatim** [comment]
Correct. Don't want to create competition between the python meetups 😀

**Comment by betatim** [comment]
Rackspace got back to us https://twitter.com/Rackspace/status/800718020293382144 so we now have two "maybes" for rooms.

**Comment by betatim** [comment]
Which reminds me, any interest to be the second speaker anyone?

**Comment by aktion-hip** [comment]
Good news concerning the room issue. I successfully booked the room HG E 41 on Di, 6. Dezember 2016, Zeit: 19-22 h.
The room is right in the corner of Rämistrasse/Karl-Schmid-Strasse (entrance level, see [ETH main building ground plan](http://www.rauminfo.ethz.ch/Rauminfo/grundrissplan.gif?region=Z&areal=Z&gebaeude=HG&geschoss=E&raumNr=41)).

**Comment by TitusCornelius** [comment]
great news, thanks!!

**Comment by uweschmitt** [comment]
Organized a group account for wifi:

> Login: pydata
> Passwort:****
> von: 06.12.2016  um: 18:30
> bis: 06.12.2016  um: 23:00

**Comment by TitusCornelius** [comment]
great, thanks! That is very nice of you :)

**Comment by uweschmitt** [comment]
Thank @aktion-hip, he suggested that, he is the ETH infrastructure pro :)

**Comment by betatim** [comment]
Rumour has it the front door to the ETH building locks at 7pm? If yes we should post on the meetup page to let people know that they should arrive 5-10min early.

**Comment by betatim** [comment]
Is there any restriction from ETH on us bringing drinks and nibbles like last time?

**Comment by aktion-hip** [comment]
I'm gonna clarify this and will provide the proper information.

**Comment by aktion-hip** [comment]
@betatim 
Concerning the opening hours of the ETH main building, there’s no problem. The building is open from 6am to 10pm (see [Öffnungszeiten HG](https://www.ethz.ch/content/dam/ethz/associates/services/Service/betrieb/flyer_isc/hg_spez/hauptgebaeude-oeffnungszeiten-feiertage.pdf)).
Concerning the drinks and nibbles: this should be no problem, given the room is left in the same state it was before.
Another note: The room HG E 41 is occupied up to 7pm. Therefore, we can enter it at the earliest on 7pm.

**Comment by uweschmitt** [comment]
I have to cancel my talk, I'm sick. Could not prepare anything today and will not come tomorrow. How do we handle the room ?

Sorry for the late notice.

**Comment by betatim** [comment]
Yes and it was super! :) Thanks for organising this.

**Comment by uweschmitt** [comment]
How many came ?

**Comment by aktion-hip** [comment]
I got the following complaint:
==
Sehr geehrter Herr Luthiger

Ich weiss nicht, ob ich bei Ihnen an der richtigen Adresse bin. Gemäss Raumbuchungsprogramm haben Sie gestern Abend als letzter den Raum HG E41 genutzt (ID WCMS, LMS & Mobile Applications).
Meine Mitarbeiter fanden heute Morgen eine grössere Anzahl leerer Bierdosen in dem Raum vor (siehe Foto unten). 
Ich möchte Sie darauf hinweisen, dass das Konsumieren von Speisen und Getränken in den Schulungsräumen untersagt ist. 

Freundliche Grüsse
Christoph Schumann
==
Can you comment on this?

**Comment by uweschmitt** [comment]
@aktion-hip I'm very sorry. I promised to to take care and was to late to find somebody who could care about the room. See my post above.

**Comment by aktion-hip** [comment]
@uweschmitt Yes, I noticed that and I don't blame you.

**Comment by betatim** [comment]
:( Sorry about that. We made an effort to leave the room in the state we found it in etc but didn't realise about the no drinking/eating rules. Is there something we can do to help fix this?

**Comment by aktion-hip** [comment]
@betatim Well, it’s a little bit unfortunate to leave these traces. But I see, the cause was that we (the ETH people) could not get it to this meetup and take our share of the responsibility.

However, I think there’s no need to overvalue this if it’s only two bags of empty beer cans (according to the photo I got). I guess I can handle this (i.e. iron it out ;-)).

**Comment by uweschmitt** [comment]
😅

**Comment by TitusCornelius** [comment]
it was only the two trashbags and we didn't know that it was not allowed to
eat and drink.. we left everything tidy and of course we would have acted
differently if we would have known.

I just went to check and take them away in case, but they re not there
anymore..

Sorry again and thanks for letting us have the room!

@uweschmitt hope you get well soon and sorry again for the hassle!

On Wed, Dec 7, 2016 at 10:57 Uwe Schmitt <notifications@github.com> wrote:

😅

—
You are receiving this because you commented.


Reply to this email directly, view it on GitHub
<https://github.com/pydatazrh/meetup/issues/7#issuecomment-265404701>, or mute
the thread

**Comment by aktion-hip** [comment]
@TitusCornelius 

> ... and we didn't know that it was not allowed to eat and drink ...

Yes, I missed to communicate that.
I just phoned to the ETH facility people and they confirmed that they have no further complaints.
Therefore, I hope we can do further meetups at the ETH if desired.
